### DISCUSSION TITLE
Hide only the picture icon

### DISCUSSION THREAD

**Discussion by laguinch** [photobooth-app/photobooth-app#99] [Q&A]
Hy, is it possible to disable the collage icon on the panel like the admin icon ? Thx

**Comment by mgineer85** [comment]
Not yet - will also be added to next release!

**Reply by laguinch** [reply]
Ok thx a wait this release with impatience 🤞🏻

**Comment by mgineer85** [comment]
Updated version 0.10.0 released.
